src/main.cpp

```diff
@@ -1,105 +1,106 @@
-#include <Arduino.h>
-#include <NativeEthernet.h>
-#include <NativeEthernetUdp.h>
+#include "EthernetDriver.h"
 
 #include "pb_encode.h"
 #include "pb_decode.h"
 #include "urc.pb.h"
 
-IntervalTimer encoderMessageTimer;
-elapsedMillis elapsedTime;
-
-byte mac[] = { 0xDE, 0xAD, 0xBE, 0xEF, 0xFE, 0xED };
-IPAddress ip(192, 168, 8, 255);
-constexpr uint16_t localPort = 8888;
-EthernetUDP Udp;
+// Function Declarations
+void setFlag();
 
+IntervalTimer timer;
 volatile bool printFlag = false;
 
-void sendEncoderEstimates(DriveEncodersMessage driveEncodersMessage);
-void updateFlag();
-void connectUdp();
+using namespace ethernet_driver;
```

Comment:
This is generally bad practice for C++. Use namespace aliases if you want to reduce the amount of typing overhead

Reply:
Fixed